Sources/NineSunsSDK/Authentication/NSAuthentication.swift

```diff
@@ -43,8 +43,12 @@ public struct NSAuthentication {
     }
     let accountInfo = NSAccountInfo(
       auth: accountAuth,
-      encryptedClientData: resultProof.data.encryptedClientData,
-      passwordDerivedSecret: passwordDerivedSecret
+      passwordDerivedSecret: passwordDerivedSecret,
```

Comment:
Should we remind the developer who using this Swift SDK this passwordDerivedSecret should only be stored under user's local device and must not be exposed to the Internet? Can we store this value to the apple keychain automatically and not return to the user?

Reply:
In the SDK scope, we cannot and shouldn't save this value to keychain. But I think we can remind the developer about the `passwordDerivedSecret` as your suggestion.

Reply:
Hi @spartan-nguyentruong , I updated the SDK to include the description and we may remind them later in the README file as well.